backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/process/adprocess/ADProcessPostProcessService.java

```diff
@@ -400,12 +396,9 @@ else if (ViewOpenTarget.NewBrowserTab.equals(target))
 		else if (recordsToOpen != null && recordsToOpen.getTarget() == OpenTarget.SingleDocument)
 		{
 			final DocumentPath documentPath = extractSingleDocumentPath(recordsToOpen);
-			final boolean openInNewTab = recordsToOpen.isOpenInNewTab();
 
 			return OpenSingleDocument.builder()
 					.documentPath(documentPath)
-					.modal(false)
-					.openInNewTab(openInNewTab)
```

Comment:
pls precisely set the targetTab, even if we have a default defined.
This will make the code more readable and robust.